pandas/core/frame.py

```diff
@@ -274,14 +274,6 @@
 4  bar           2  bar            6
 5  baz           3  baz            7
 
-Merge DataFrames df1 and df2, but raise an exception if the DataFrames have
-any overlapping columns.
-
->>> df1.merge(df2, left_on='lkey', right_on='rkey', suffixes=(False, False))
```

Comment:
did we previously accept False here? or was this added in the previous PR?

Reply:
this was not added in previous PR.
IIUC previously the code was something like
` if not lsuffix and not rsuffix`, so `False` and `None` are both not allowed, and will raise the `ValueError`,
but since in this PR, we don't raise error or warnings when there are duplicate column names, so i removed this part of docstring. @jreback @jorisvandenbossche